cardstack.code-workspace

```diff
@@ -0,0 +1,43 @@
+{
+  "folders": [
+    {
+      "path": "./packages/cardpay-reward-api"
+    },
+    {
+      "path": "./packages/cardpay-reward-indexer"
+    },
+    {
+      "path": "./packages/cardpay-reward-programs"
+    },
+    {
+      "path": "./packages/reward-root-submitter"
+    }
+  ],
+  "settings": {
+    "[python]": {
+      "editor.formatOnSave": true,
+      "editor.codeActionsOnSave": {
+        "source.organizeImports": true
+      }
+    },
+    "python.defaultInterpreterPath": "/opt/homebrew/Cellar/pdm/1.15.3/libexec/bin/python3.10",
```

Comment:
I thnk theres no way around this. Must just specify the interpreter path given by `pdm use`

Reply:
Resolved. Just have to specify python path in user space settings. Include in readme to use an interpreter of python3.10.